services/docsrs/docsrs.service.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const Joi = require('joi')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.array()
+  .items(
+    Joi.object({
+      build_status: Joi.boolean().required(),
+    })
+  )
+  .min(1)
+  .required()
+
+module.exports = class DocsRs extends BaseJsonService {
+  static category = 'build'
+  static route = { base: 'docsrs', pattern: ':crate/:version?' }
+  static examples = [
+    {
+      title: 'docs.rs',
+      namedParams: { crate: 'regex', version: 'latest' },
+      staticPreview: this.render({ version: 'latest', buildStatus: true }),
+      keywords: ['rust'],
+    },
+  ]
+
+  static defaultBadgeData = {}
+
+  static render({ buildStatus, version }) {
+    if (buildStatus) {
+      return {
+        label: `docs@${version}`,
+        message: 'passing',
+        color: 'success',
+      }
+    } else {
+      return {
+        label: `docs@${version}`,
+        message: 'failing',
+        color: 'critical',
+        isError: true,
```

Comment:
Let's remove this, as it's used to reflect an error within Shields/processing/etc., not that the upstream build failed.

```suggestion
```